### DISCUSSION TITLE
增加Waline评论的”tag“选项

### DISCUSSION THREAD

**Discussion by Lfhsheng** [YunYouJun/hexo-theme-starry#232] [Q&A 疑难杂症] [answered]
如题，我想把我的Waline`博主`的`tag`换成`舰长`

换个方法问，如何更改或增加`tag`

https://github.com/walinejs/waline/pull/118

**Comment by YunYouJun** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/25154432/148350473-333f7cf2-380f-4e2a-a59b-f38102510609.png)

对应配置 YAML 中 Waline 字段即可。

**Comment by Lfhsheng** [comment]
给各位提示一下
```yaml
waline: 
  langMode:
    admin: '舰长'
```

**Comment by Lfhsheng** [comment]
https://waline.js.org/reference/client.html#langmode

各位可以使用`locale`
